sycl/test-e2e/Matrix/joint_matrix_apply_two_matrices.cpp

```diff
@@ -0,0 +1,14 @@
+//==------ joint_matrix_apply_two_matrices.cpp  - DPC++ joint_matrix--------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: aspect-ext_intel_matrix
```

Comment:
I see this test is not running in pre-ci on ARC. I believe the reason is that we need to add matrix-xmx8, just aspect is not enough for XMX8 tests. Something like this (not sure the exact format)?
```suggestion
// REQUIRES: aspect-ext_intel_matrix || matrix-xmx8
```

Reply:
This is a problem as we want to remove matrix-xmx8 in the future.
is this the case for all the matrix tests we recently added?
Who can help here?

Reply:
I was wrong about requirement to add matrix-xmx8. It seems we just have some problem in lit.cfg.py and/or CI system configuration, so ext_intel_matrix is not currently visible and hence this and other tests do not run in pre-commit in SYCL.